nodeJS/express_and_mongoose/deployment.md

```diff
@@ -95,22 +95,22 @@ Whatever your circumstances, we've got you covered. Here are the PaaS providers
 
 #### Fly.io
 
-- Fly.io uses a convenient CLI tool for deployment.
-- All plans include [free allowances](https://fly.io/docs/about/pricing/#free-allowances), additional resources are billed based on usage with very reasonable rates. Each app should cost around $4 per month.
-- $20 a month should be enough to host eight apps (including three apps for free).
+- Fly.io uses a CLI tool for deployment.
+- Paid plans include [free allowances](https://fly.io/docs/about/pricing/#free-allowances), additional resources are billed based on usage with very reasonable rates.
 - Fly.io charges $0.15/GB of RootFS for machines stopped for 30 days.
 
 ##### Fly.io: Hobby Plan
 
-- New customers get a one-time $5 free trial credit to test Fly.io at no cost. The free credit will only start being used after the free allowances are exceeded. After the credit has been used, you will be be automatically placed on the $5/month Hobby plan subscription.
-- The longevity of your free trial credit depends on how many resources you consume. More complex apps with more traffic may consume all free resources within a month, whereas simpler apps may last longer.
+- New customers get a one-time $5 free trial credit to test Fly.io at no cost. The free allowances are not applicable during the free trial. After the credit has been used, you will be be automatically placed on the $5/month Hobby plan subscription.
+- The longevity of your free trial credit depends on how many resources you consume. More complex apps with more traffic may consume the whole free trial credit within a month, whereas simpler apps may last longer.
 - Requires a credit card.
 
 ##### Fly.io: Links
 
-- [Fly.io's homepage](https://fly.io/)
-- [Fly.io's documentation](https://fly.io/docs/)
-- [Guide: Official getting started with NodeJS on Fly.io guide](https://web.archive.org/web/20230823151155/https://fly.io/docs/languages-and-frameworks/node/)
+- [Homepage](https://fly.io/)
+- [Documentation](https://fly.io/docs/)
+- [Guide: Official getting started with Rails on Fly.io guide](https://fly.io/docs/rails/getting-started/)
+- [Guide: GoRails video guide for deploying Rails applications to Fly.io](https://www.youtube.com/watch?v=6Zp9y8nF5rE&)
```

Comment:
You've accidentally copied over the links from the Rails deployment file. These will need to be reverted in the Node version as below. The rest of the changes look fine to me.
```suggestion
- [Fly.io's homepage](https://fly.io/)
- [Fly.io's documentation](https://fly.io/docs/)
- [Guide: Official getting started with NodeJS on Fly.io guide](https://web.archive.org/web/20230823151155/https://fly.io/docs/languages-and-frameworks/node/)
```